### DISCUSSION TITLE
Errno 13 - Ubuntu - Systemd daemon

### DISCUSSION THREAD

**Discussion by palmy-investing** [celery/django-celery-beat#644] [Q&A] [answered]
Hi whats good. I am currently trying to start celerybeat as a systemd daemon. The daemon restarts and restarts with nearly 100% cpu usage. 

I checked the logs; errno 13 for ‚celery-beat-schedule’ pointing to no where (at least not a file since errno 13 is about perms on the server level). 

So my question is: What permission does the user who run the damon need? Does he need all perms on celery.py (beat-task conf) & all for settings.py so rwx? In settings.py all namespaces with CELERY are used for configs

I already tried it without the —pidfile, —logfile, but they seem to work since the failure remains. 

Which files are accessed exclusively by beat which arent used for the normal celery worker? 

Ive followed the official instructions but I am pretty unsure where this error comes from.

**Comment by palmy-investing** [comment, marked as answer]
Everyone with errno 13 make sure to set -s to a valid dir. For me it worked after updating the -s path

**Reply by auvipy** [reply]
always open discussion for support issues
